src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/listeners/MigrationMessageType.kt

```diff
@@ -11,4 +11,5 @@ enum class MigrationMessageType {
 
 enum class SynchronisationMessageType {
   RETRY_SYNCHRONISATION_MAPPING,
+  RETRY_NON_ASSOCIATION_MAPPING,
```

Comment:
minor but I think RETRY_SYNCHRONISATION_MAPPING is the exact message you want, so no need to define a new message

Reply:
thanks - will remove